### PR TITLE
Bible tracker testing bug fixes and enhancements

### PR DESCRIPTION
**Pull request by czhaocmu**
- #334 
- Url parameters to open to certain week for announcements (?week=13)

### CODE REVIEW

File: ap/bible_tracker/views.py [hunk 1]
```diff
@@ -113,6 +112,7 @@ def index(request):
     weekly_reading = trainee_bible_reading.weekly_reading_status[term_week_code]
     json_weekly_reading = json.loads(weekly_reading)
     weekly_status = str(json_weekly_reading['status'])
+    finalized = str(json_weekly_reading['finalized'])
   else:
     weekly_status = "_______"
```
Comment (jontey) [inline ap/bible_tracker/views.py]: You forgot to set the finalized string here for the case where the trainee has not input anything yet

**Comment by czhaocmu** [comment]
@jontey I added more changes to this branch for TA view and finalization etc.

File: ap/bible_tracker/templates/bible_tracker/report.html [hunk 2]
```diff
@@ -147,6 +207,54 @@
       <br>
   <div class="data-table-container">
 
+<<<<<<< HEAD
```
Comment (jontey) [inline ap/bible_tracker/templates/bible_tracker/report.html]: This too...

File: ap/bible_tracker/views.py [hunk 3]
```diff
@@ -207,9 +230,57 @@ def updateStatus(request):
...
     hstore_weekly_reading = json.dumps(json_weekly_reading)
     trainee_bible_reading.weekly_reading_status[term_week_code] = hstore_weekly_reading
     trainee_bible_reading.save()
 
-    return HttpResponse(weekly_status)
\ No newline at end of file
+    return HttpResponse(weekly_status)
+
+def finalizeStatus(request):
+  my_user = request.user;
```
Comment (jontey) [inline ap/bible_tracker/views.py]: Python doesn't use semi-colons for line endings :)

File: ap/bible_tracker/views.py [hunk 4]
```diff
@@ -207,9 +230,57 @@ def updateStatus(request):
...
+  if request.is_ajax():
+    action = request.POST['action']
+    #if not TA, cannot finalize till right time.
```
Comment (jontey) [inline ap/bible_tracker/views.py]: Line 247 to 252 should be after line 257

**Pull request merged**